gradle/libs.versions.toml

```diff
@@ -3,22 +3,22 @@
 [versions]
 accompanist = "0.28.0"
 agp = "8.1.1"
-androidCompileSdk = "33"
+androidCompileSdk = "34"
 androidMinSdk = "21"
 androidTargetSdk = "32"
 androidx-lifecycle = "2.6.1"
 androidx-navigation-compose = "2.5.1"
 coil = "2.2.1"
-composePlugin = "1.4.0"
+composePlugin = "1.5.1"
 composeBom = "2023.05.01"
-composeCompiler = "1.4.4"
+composeCompiler = "1.5.3"
 coroutines = "1.6.4"
 dependencyAnalysis = "1.13.1"
 detekt = "1.21.0"
 junit5 = "5.8.2"
 jvmTarget = "11"
-kotlin = "1.8.10"
-ksp = "1.8.0-1.0.8"
+kotlin = "1.9.10"
```

Comment:
Too early, we might not be able to use this version internally.

Reply:
I know, was testing a fix for an issue https://youtrack.jetbrains.com/issue/KT-60230/Native-unknown-options-iossimulatorversionmin-sdkversion-with-Xcode-15-beta-3 which was fixed in 1.9.10

Reply:
You can set xcode version to the one that is currently supported (idk which one on 1.8, just try)
https://github.com/badoo/Reaktive/blob/master/.github/workflows/build.yml#L27